spec/lib/rmt/cli/main_spec.rb

```diff
@@ -170,5 +170,16 @@
         end
       end
     end
+
+    describe 'Thor default behaviour when errors' do
+      let(:argv) { ['asdasdas'] }
+      let(:error_message) { "Could not find command \"asdasdas\".\n" }
+
+      it 'exits with status 1' do
+        expect { command }
+          .to raise_error(SystemExit)
```

Comment:
Exit code isn't actually being tested here, despite what the example name suggests.

```suggestion
          .to raise_error(SystemExit) { |e| expect(e.status).to eq 1 }
```

Reply:
:+1: